spec/javascript/components/lease_ups/LeaseUpsPage.test.js

```diff
@@ -40,7 +40,7 @@ describe('LeaseUpApplicationsPage', () => {
 
   test('Should render LeaseUpTable', () => {
     const applications = [ buildApplicationPreference(1), buildApplicationPreference(2) ]
-
+    console.log("PLEASE LOG THIS", applications)
```

Comment:
@akegan the message is pretty clear, but do we want this console.log in here? why?

Reply:
nope, it's just a debug. I'll remove it

Reply:
I was just really hoping that I would see the log 🤣